### ISSUE TITLE
Payments not going through

### ISSUE THREAD

**Issue by easye35** [bitcoinxt/bitcoinxt#177]
I've made several bitcoin payments the past few weeks, all of them get stuck. the first two only went through on the third day.  the payment I made yesterday is stuck unconfirmed.

Status: 0/unconfirmed
Date: 12/1/2016 09:50
To: 16UYQwUqEmExAWa8bBS6RMrjawg6FLCncr
Debit: -0.00100000 BTC
Transaction fee: -0.00011695 BTC
Net amount: -0.00111695 BTC
Transaction ID: 436b56f6d2a8346e971088abaddc3ad97f8982ded050475159d91f13b50a86cf-000

it's been over 24 hours, I can't see why it is stuck. I don't know where I should be asking for help on this, but I think it's the core, as I made three payments with same results, all to different address's.  all from the same wallet/computer

**Comment by dagurval** [comment]
Did you use the default fee rate suggested by Bitcoin XT? If so, we should bump the default fee. Due to broken network conditions (full blocks) the fee required is absurdly high.

If your transaction does not confirm within 72 hours, the network may allow you to re-send them (double spend them) with a higher fee.

To do this:
Backup of your wallet. Then use `-zapwallettxes` to remove unconfirmed transactions from your wallet. This will cause a full block rescan, so this will take some time. Then re-send your transactions with a high fee.

Fee estimation:
https://bitcoinfees.21.co/

How to use zapwallettxes:
http://bitcoin.stackexchange.com/questions/45988/how-do-you-apply-zapwallettxes-on-windows

**Comment by dagurval** [comment]
Related issue: https://github.com/bitcoinxt/bitcoinxt/issues/174

**Comment by easye35** [comment]
Yes, I used default fee. Last time it waited either close to or right at 72
hours, then confirmed all at once

**Comment by easye35** [comment]
issue #174 from what I'm reading is that the tans fee is setting to a default amount, but my trans fee was 0.00011695 BTC which is way about what that complaint was about I think.

**Comment by easye35** [comment]
I just sent another payment, this time it at least says broadcast through 8 nodes, I notice the stuck one doesn't say that. is that the problem? It isn't getting accepted by the blockchain?

**Comment by dagurval** [comment]
For anyone else having this issue, ViaBTC has created a workaround transactions that are not confirming: https://www.viabtc.com/tools/txaccelerator/
